content/talks/containers.md

```diff
@@ -14,6 +14,10 @@ tags:
   - Container Queries
   - CSSWG
 events:
+  - venue: CMD Amsterdam
+    date: 2024-06-05
+    adr: Amsterdam, Netherlands
+    slides: https://slides.oddbird.net/cq/pre-cssday/
```

Comment:
Do we have a URL to add here?

Reply:
We could add the eventbrite link: https://www.eventbrite.nl/e/pre-css-day-tickets-872463962197

Reply:
```suggestion
    url: https://www.eventbrite.nl/e/pre-css-day-tickets-872463962197
    slides: https://slides.oddbird.net/cq/pre-cssday/
```